src/test/ValidationTest.java

```diff
@@ -0,0 +1,26 @@
+package test;
+
+import org.junit.Assert;
+import org.junit.Test;
+import validation.Validation;
+
+public class ValidationTest {
+
+    @Test
+    public void checkBracketsTestOK(){
+        boolean actualResult = Validation.checkBrackets("(data()Art())");
+        Assert.assertTrue(actualResult);
+    }
+
+    @Test
+    public void checkBracketsTestInvalid(){
+        boolean actualResult = Validation.checkBrackets("(data(Art())");
+        Assert.assertTrue(!actualResult);
```

Comment:
@YatKat please use assert false
```suggestion
        Assert.assertFalse(actualResult);
```